packages/kotti-ui/source/kotti-style/_comments.scss

```diff
@@ -1,127 +1,128 @@
-.comment {
+.kt-comment {
 	display: flex;
 	flex-flow: row;
 	padding: 0.4rem 0;
 	border-bottom: 1px solid #dbdbdb;
-}
-
-.comment__avatar {
-	flex: 0 0 2.4rem;
-	height: 2.4rem;
-}
-
-.comment__wrapper {
-	display: flex;
-	flex: 1 1 auto;
-	flex-direction: column;
-	margin-left: 0.4rem;
-}
-
-.comment__info {
-	display: flex;
-	flex-direction: row;
-	width: 100%;
-	font-size: 0.7rem;
-	line-height: 1.2rem;
-}
-
-.info__name {
-	flex: 1 1;
-	font-weight: 600;
-}
-
-.info__time {
-	flex: 1 1;
-	color: $lightgray-500;
-	text-align: right;
-}
-
-.comment__message {
-	font-size: 0.75rem;
-	line-height: 1.2rem;
-	word-break: break-word;
-}
-
-.comment__action,
-.comment-reply__action {
-	display: flex;
-	flex-direction: row;
-	justify-content: space-between;
-	margin: 0.2rem 0;
-	font-size: 0.7rem;
-	font-weight: 600;
-	line-height: 1.2rem;
-	color: $lightgray-500;
-	.yoco {
-		font-size: 0.9rem;
+	&__wrapper {
+		display: flex;
+		flex: 1 1 auto;
+		flex-direction: column;
+		margin-left: 0.4rem;
+	}
+	&__info {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		font-size: 0.7rem;
+		line-height: 1.2rem;
+		&__name {
+			flex: 1 1;
+			font-weight: 600;
+		}
+		&__time {
+			flex: 1 1;
+			color: $lightgray-500;
+			text-align: right;
+		}
+	}
+	&__message {
+		font-size: 0.75rem;
+		line-height: 1.2rem;
+		word-break: break-word;
+	}
+	&__action {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		margin: 0.2rem 0;
+		font-size: 0.7rem;
+		font-weight: 600;
+		line-height: 1.2rem;
+		color: $lightgray-500;
+		.yoco {
+			font-size: 0.9rem;
+		}
+		&__options {
+			position: absolute;
+			top: 1.2rem;
+			right: 0;
+			z-index: $zindex-4;
+			display: none;
+			height: auto;
+			padding: 0.4rem 0;
+			line-height: 1.4rem;
+			text-align: left;
+			background: #ffffff;
+			border-radius: $border-radius;
+			box-shadow: $box-shadow;
+			li {
+				padding: 0.2rem 1rem;
+				list-style: none;
+			}
+			li:hover {
+				background: $lightgray-300;
+			}
+		}
+		&__reply {
+			display: flex;
+			flex: 0 0 auto;
+			align-items: center;
+			padding: 0 0.2rem;
+			border-radius: 0.2rem;
+			&:hover {
+				color: $darkgray-400;
+				cursor: pointer;
+				background: $lightgray-400;
+			}
+		}
+		&__more {
+			position: relative;
+			display: inline-block;
+			flex: 0 0 2rem;
+			padding: 0 0.2rem;
+			text-align: center;
+			border-radius: 0.2rem;
+			&:hover {
```

Comment:
Please leave newlines in front of nesting for readability. Unfortunately, the autoformatter doesn’t handle this yet.